deployments/spacelift/dpe-k8s/main.tf

```diff
@@ -217,41 +217,44 @@ resource "spacelift_aws_integration_attachment" "k8s-deployments-aws-integration
 }
 
 
-resource "spacelift_stack" "auth0" {
-  github_enterprise {
-    namespace = "Sage-Bionetworks-Workflows"
-    id        = "sage-bionetworks-workflows-gh"
-  }
-
-  depends_on = [
-    spacelift_space.dpe-space
-  ]
+# resource "spacelift_stack" "auth0" {
+#   count = 0
+#   github_enterprise {
+#     namespace = "Sage-Bionetworks-Workflows"
+#     id        = "sage-bionetworks-workflows-gh"
+#   }
```

Comment:
To be reverted before merge. We likely want to look to modify this eventually to conditionally create this `auth0` stack depending on if authentication is required for the stack. Currently it'll always be deployed, and, for the sandbox stack for Brad, we aren't going to be exposing the sandbox to the internet.